README.md

```diff
@@ -79,6 +79,14 @@ deploy either the survey runner or the author follow these steps:
 1. eb list (will give you the environment name)
 1. eb deploy <your-env-name>
 
+## Alerting
+A webhook will need to be created for a new integration via https://api.slack.com/incoming-webhooks
+Alternatively, you team may already have a webhook url created to send messages to your slack.  
+
+You will need to create a slack channel with the name, eq-<your-env-name>-alerts
```

Comment:
The `<your-env-name>` doesn't appear in the rendered readme:
![image](https://cloud.githubusercontent.com/assets/1754822/24146909/c96cf56c-0e2f-11e7-81a6-ca5f0b440450.png)

It would be better to put it all in backticks - `eq-<your-env-name>-alerts`